packages/components/src/ComboBox.tsx

```diff
@@ -223,21 +239,19 @@ class ComboBox extends Component<ComboBoxProps, ComboBoxState> {
     switch (direction) {
       case ComboBox.MENU_NAVIGATION_DIRECTION.UP:
         if (keyboardOptionIndex > 0) {
```

Comment:
There's a clever way to write this kind of thing, so you can add and mod correctly without having an else case for the wrap. Just add the menuOptionsLength after subtracting, something like:
```
let delta = 0;
switch ComboBox.Up:
  delta = -1;
  break;
switch ComboBox.Down:
  delta = 1;
  break;

if (delta != 0) {
  this.setState({ keyboardOptionIndex = (keyboardOptionIndex + delta + menuOptionsLength) % menuOptionsLength })
}
```

Reply:
I switched it to this way